io-functions-pay-portal/utils/responses.ts

```diff
@@ -63,6 +65,12 @@ export const toErrorServerResponse = <S extends number, T>(
       return ResponseErrorNotFound("Not found", "Resource not found");
     case 429:
       return ResponseErrorTooManyRequests("Too many requests");
+    case 500:
+      return ResponseErrorInternal(
+        PaymentProblemJson.decode(response.value).getOrElse({
+          detail: PaymentFaultEnum.PAYMENT_UNKNOWN
+        }).detail || "PaymentProblemJson on decode"
```

Comment:
This is a default error, for example when the _proxy-node_ returns 500 without a body, maybe I would change to "Generic Error".